scripts/install-mender.sh

```diff
@@ -466,7 +466,14 @@ do_setup_mender_client() {
 
     echo "  Setting up mender with options: $MENDER_SETUP_ARGS"
     `mender_setup_cli` $MENDER_SETUP_ARGS
-    pidof systemd && systemctl restart mender-client
+    pidof systemd && (
+        if [[ "$SELECTED_COMPONENTS" == *"mender-client4"* ]]; then
```

Comment:
This won't work if, for example, I "select" `mender-update` for install by running `get.mender-io ... mender-update`. (Didn't we fix this elsewhere?)

We should instead check for all the services individually. What about:
```
systemctl status mender-client && systemctl restart mender-client
systemctl status mender-updated && systemctl restart mender-updated
systemctl status mender-authd && systemctl restart mender-authd
```

Reply:
@lluiscampos please properly check the function: you reach this point only if you are installing mender-client or mender-client4.

Reply:
Thanks for pointing that out,

Then, it is that [previous check](https://github.com/mendersoftware/mender-dist-packages/pull/433/files#diff-4f0df1d73eedfab863e987b0c7f29079133c80b39e5e68b15eee877095dc612fR458) that will misbehave in the scenario that I was describing: if I install `mender-update` directly, it will pull `mender-setup` (recommended package) and then this function should still run if I have specified setup arguments.

Reply:
ok, let's do it that way